js/modules/treemap.src.js

```diff
@@ -246,20 +279,67 @@ seriesType(
         alternateStartingDirection: false,
 
         /**
-         * Used together with the levels and allowDrillToNode options. When
-         * set to false the first level visible when drilling is considered
-         * to be level one. Otherwise the level will be the same as the tree
-         * structure.
+         * Used together with the levels and allowTraversingTree options. When
+         * set to false the first level visible to be level one, which is
+         * dynamic when traversing the tree. Otherwise the level will be the
+         * same as the tree structure.
          *
          * @since 4.1.0
          */
         levelIsConstant: true,
-
         /**
          * Options for the button appearing when drilling down in a treemap.
+         * Deprecated and replaced by [traverseUpButton](#plotOptions.treemap.traverseUpButton).
+         *
+         * @deprecated
          */
         drillUpButton: {
 
+            /**
+             * The position of the button.
+             *
+             * @deprecated
+             */
+            position: {
+
+                /**
+                 * Vertical alignment of the button.
+                 *
+                 * @default    top
+                 * @validvalue ["top", "middle", "bottom"]
+                 * @product    highcharts
+                 * @apioption  plotOptions.treemap.traverseUpButton.position.verticalAlign
+                 */
```

Comment:
It is actually not quite a duplicate, it is the old option which has been renamed and is now deprecated. If I remove it then it will not appear in the api, which I would like so that I can inform of the deprecation.
The best thing would be to make a alias/copy in the options and just set one of them to deprecated, but I do not believe this is possible with current system.

Reply:
Isn't the `apioption` the same (311 vs 354), so both will create the same API doc?

Reply:
You are correct, these options were the same ones, but they should not be. I corrected this in dfab9a4 and deprecated the one in `drillUpButton`.